organization/governance-model.md

```diff
@@ -0,0 +1,71 @@
+---
+type: Resource
+---
+
+# Governance model for the Foundation for Public Code
+
+This provides a non-legal summary of the Foundation's governance model as established in the [association bylaws](association-bylaws.nl.md).
+
+## Overview
+
+The Foundation for Public Code is a [member-owned association (vereniging) under Dutch law](https://business.gov.nl/starting-your-business/choosing-a-business-structure/association/). Only wholly publicly owned organizations may become members.
+
+This means members are responsible for and steer the Foundation's operations, including priorities and annual budget for the organization.
+
+## Foundation bodies
+
+The Foundation consists of four bodies:
+
+* general assembly
+* board of directors
+* advisory board
+* secretariat
+
+### General assembly
+
+All members are represented in the general assembly.
+
+The general assembly:
+
+* approves the Foundation's operational plan for the coming year
+* elects the board of directors
+
+These decisions are agreed through a majority vote. Voting rights are proportional to members' membership dues. This ensures democratic ownership and control by all members.
+
+The general assembly meets at least once every year. Minutes from the general assembly meetings are shared with all members.
+
+We commit to holding all meaningful votes electronically, so that all members can be represented regardless of ability to attend physically.
+
+### Board of directors
+
+The board of directors is elected by the general assembly. By law, it must include a chair, a secretary and a treasurer. The general assembly can vote to add more roles to the board of directors.
+
+The board of directors:
+
+* proposes the Foundation's operational plan and budgeting for the coming year
+* approves membership applications
+
+These decisions are agreed upon through majority vote.
+
+The board of directors meets regularly and is required to keep the general assembly informed about its activities.
+
+The current board members are:
+
+* Ben Cerveny (Chairman)
+* Arnout Schuijff (Treasurer)
+* Boris van Hoytema (Secretary)
+* Pieter van der Does
+
+### Advisory board
+
+The advisory board is a set of experts that suppport the general assembly, board of directors and the secretariat. There are some decisions for which they must be consulted.
+
+### Secretariat
```

Comment:
We spoke in some configuration about not calling it the secretariat, and instead just identifying it as the staff. This would mean the staff are less institutionally aligned with the Secretary.

Reply:
Yes, I'm still doubting whether this should be blocking the release...

I'd say the staff falls under the board of directors, not necessarily the secretariat.

Reply:
We've deliberately left out all information about the relationship between the staff and the directors (and especially which decisionmaking is delegated from the directors to the staff) because:
a) it's not specifically covered in the akte van oprichting
b) we'll only formally know after the first board of director's meeting

Reply:
Exactly that is what the change to remove the word 'secretariat' does right, otherwise, it specifies a relationship.

In my view the relationship has always meant like this, me also being the secretary as well as the chief executive doesn't change that.